src/AspNet.Security.OAuth.Introspection/Events/AccessTokenReceivedContext.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.AspNetCore.Http;
+
+namespace AspNet.Security.OAuth.Introspection {
+    public class AccessTokenReceivedContext : BaseIntrospectionContext {
+        public AccessTokenReceivedContext(
+            HttpContext context, 
```

Comment:
True... Except when ReSharper decides to be a butt and throw RED UNDERLINES ALL OVER even though nothing's wrong with your code or references... :angry:

But I digress. I'll add them :smile_cat: